core/src/apps/cardano/shelley/bech32.py

```diff
@@ -0,0 +1,14 @@
+from trezor.crypto import bech32
+
+
+def bech32_encode(hr_part, b):
+    convertedbits = bech32.convertbits(b, 8, 5)
```

Comment:
I don't understand what "hr" stands for  (header?). Let's use a longer, more descriptive name, and keep it constistent in both encode and decode.